### PR TITLE
// Fix input number on product page - BOOM-323

### PR DESCRIPTION
**Pull request by nihco2**

### CODE REVIEW

File: themes/classic/templates/catalog/product.tpl [hunk 1]
```diff
@@ -264,7 +264,7 @@
                             {block name='product_quantity'}
                               <p class="product-quantity _margin-top-medium">
                                 <label for="quantity_wanted">{l s='Quantity'}</label>
-                                <input type="number" min="1" name="qty" id="quantity_wanted" value="{$product.quantity_wanted}" class="input-group" />
+                                <input type="text" min="1" name="qty" id="quantity_wanted" value="{$product.quantity_wanted}" class="input-group" />
```
Comment (mickaelandrieu) [inline themes/classic/templates/catalog/product.tpl]: if your input is now a test,  the `min` attribute does not exists anymore:)

**Comment by nihco2** [comment]
ping @Leadesign :stuck_out_tongue_winking_eye:

**Comment by vincentbz** [comment]
Test OK

**Comment by maximebiloe** [comment]
@nihco2 could you rebase it please ?

**Comment by nihco2** [comment]
@maximebiloe done.

**Comment by mickaelandrieu** [comment]
thank you @nihco2

**Pull request merged**